hoomd/md/pytest/test_methods.py

```diff
@@ -3,393 +3,303 @@
 import numpy
 import itertools
 from copy import deepcopy
+from collections import namedtuple
+
+paramtuple = namedtuple('paramtuple',
+                        ['setup_params',
+                         'extra_params',
+                         'accepted_params',
+                         'rejected_params',
+                         'method'])
+
+
+def _method_base_params():
+    method_base_params_list = []
+    # Start with valid parameters to get the keys and placeholder values
+
+    langevin_setup_params = {'kT': hoomd.variant.Constant(2.0) }
+    langevin_extra_params = {'alpha': None, 'tally_reservoir_energy': False }
+    langevin_accepted_params = {'kT': hoomd.variant.Ramp(1, 2, 1000000, 2000000),
+                      'alpha': None, 'tally_reservoir_energy': True }
+    langevin_rejected_params = {}
```

Comment:
I don't see any class with a non-empty `rejected_params` dict. Also, the name would be more clear with something like `cls_unsettable_after_attach`.

Reply:
Yes, that is true. I initially thought it might be a good idea to also have a dict of parameters, which are immutable after attachment, in place. I removed them now.